src/examples/TTSExamplePage.tsx

```diff
@@ -0,0 +1,90 @@
+'use strict';
+import { Button, FlatList, GestureResponderEvent, StyleSheet, Text, TextInput, View } from 'react-native';
+import React from 'react';
+import Tts, { Voice } from 'react-native-tts';
+import { Example } from '../components/Example';
+import { Page } from '../components/Page';
```

Comment:
nit: Remove extra spaces between text and brackets, make sure you have ESLint turned on in your code editor so files are formatted consistently